proto/prysm/v1alpha1/beacon_chain.proto

```diff
@@ -271,7 +280,9 @@ enum SetAction {
 }
 
 // ValidatorChangeSet updates the server's list of keys on which to operate.
+// DEPRECATED: This message's endpoint is superseded by the /eth/v1/events Beacon API endpoint, there is an internal GRPC endpoint /internal/eth/v1/events
```

Comment:
`/eth/v1/events` does not stream validator info. Also same comment as for `StreamAttestations`.